apcd_cms/src/client/src/utils/dateUtil.ts

```diff
@@ -14,11 +14,19 @@ export const formatDate = (dateString: string | number | Date): string => {
   }).format(date);
 };
 
-export const formatModalDate = (dateString: string | number | Date): string => {
-  return formatDate(dateString)
-    .replace('.', '')
-    .replace('AM', 'a.m.')
-    .replace('PM', 'p.m.');
+export const formatCSTDate = (dateString: string | number | Date): string => {
+  const dateUTC = new Date(dateString);
+  const date = new Date(dateUTC.valueOf() + dateUTC.getTimezoneOffset() * 60000);
+    
+  if (isNaN(date.getTime())) {
+    return '';
+  }
+
+  return new Intl.DateTimeFormat('en-US', {
+    month: 'short',
+    day: 'numeric',
+    year: 'numeric',
+  }).format(date);
```

Comment:
This will display as UTC without any further timezone specific stuff. 
Additional change would be to rename formatCSTDate to formatUTCDate

```suggestion
export const formatCSTDate = (dateString: string | number | Date): string => {
  const dateUTC = new Date(dateString);
    
  if (isNaN(dateUTC.getTime())) {
    return '';
  }

  return new Intl.DateTimeFormat('en-US', {
    month: 'short',
    day: 'numeric',
    year: 'numeric',
    timeZone: 'UTC',
  }).format(dateUTC);
```